tools/windows_buildenv.bat

```diff
@@ -46,6 +46,7 @@ EXIT /B 0
     CALL :READ_ENVNAME
     SET BUILDENV_NAME=%RETVAL%
     SET BUILDENV_PATH=%BUILDENV_BASEPATH%\%BUILDENV_NAME%
+    SET CMAKE_TOOLCHAIN_FILE=%BUILDENV_PATH%\scripts\buildsystems\vcpkg.cmake
```

Comment:
>>%CMakeSettings% echo       "cmakeToolchain": "!BUILDENV_PATH:\=\\!\\scripts\\buildsystems\\vcpkg.cmake",

Reply:
Escape characters not correct for JSON